environments/Test-Laptop.json

```diff
@@ -41,6 +42,28 @@
         "hdfs" : {
           "dfs_replication_factor" : 1
         },
+        "realm" : "BCPC.EXAMPLE.COM",
+        "enable_kerberos": false,
+        "kerberos": {
+          "data": {
```

Comment:
Consider a scenario where a single `keytab` is generated for  `HDFS` services (`namenode`, `datanode`, `journalnode`) and a single `keytab` for `yarn` services (`resourcemanage` and `nodemanager`), in such situation we need to know which service uses which `keytab` file. If each service is always using a separate `keytab` then using hash structure keys like `namenode`, `datanode` etc can be used to generate `ketyab` file name.